doc/zmq.md

```diff
@@ -78,7 +80,7 @@ bytes).
 These options can also be provided in bitcoin.conf.
 
 ZeroMQ endpoint specifiers for TCP (and others) are documented in the
-[ZeroMQ API](http://api.zeromq.org/4-0:_start).
+[ZeroMQ API](http://api.zeromq.org/).
```

Comment:
Why change this?

Reply:
it's already centrally located in doc/dependencies.md. this removes redundant info that can get out of synch later.